client/favorites.jsx

```diff
@@ -1,23 +1,65 @@
 import React from 'react';
 import NavBar from './navBar';
 import Loader from './loader';
+import fetch from 'node-fetch';
 
-export default class SearchResults extends React.Component {
+
+export default class Favorites extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      isLoading: false
+      favoritesData: '',
+      isLoading: true
     };
+    this.handleRemove = this.handleRemove.bind(this)
   }
 
+  handleRemove(e) {
+    e.preventDefault()
+    e.target.parentElement.className = 'fade1'
+    let removalId = null
+    let removalName = null
+    const data = this.state.favoritesData
+
+
+    for (let i = 0; i < data.length; i ++) {
+      if (data[i].hotelName === e.target.parentElement.innerText) {
+        removalName = e.target.parentElement.innerText
+        break
+      }
+    }
+    for (let i = 0; i < data.length; i++) {
+      if (data[i].hotelName === removalName) {
+        removalId = data[i].hotelId
+        break
+      }
+    }
```

Comment:
You can do this in one loop. Also, it would be better to use a `data-` attribute instead of reading the text. Try giving your element `data-hotel-id={???}`.